src/store/constants.ts

```diff
@@ -0,0 +1,8 @@
+import { EGameMode } from "./types";
+
+export const GAME_MODES_FFA = [
+    EGameMode.GM_FFA,
+    EGameMode.GM_LTW_FFA,
```

Comment:
```suggestion
    EGameMode.GM_FFA,
```

Reply:
LTW is no longer an active ladder. We keep the game mode in the matchmaking code base if we ever want to enable it again. Then we'll add it back to website and website-backend.